UI/data/locale/en-US.ini

```diff
@@ -124,6 +124,12 @@ AlreadyRunning.Title="OBS is already running"
 AlreadyRunning.Text="OBS is already running! Unless you meant to do this, please shut down any existing instances of OBS before trying to run a new instance. If you have OBS set to minimize to the system tray, please check to see if it's still running there."
 AlreadyRunning.LaunchAnyway="Launch Anyway"
 
+# warning if auto safe-mode has engaged
+AutoSafeMode.Title="Safe Mode"
+AutoSafeMode.Text="OBS did not shut down properly during your last session.\n\nWould you like to start in Safe Mode and disable all third party plugins?"
```

Comment:
We could probably nitpick this text all day, but this suggests running in safe mode and disabling 3rd party plugins are two separate actions. Perhaps something like "Would you like to start in Safe Mode (disables third party plugins)?"

Reply:
This wording was suggested by Warchamp but I agree that could be clearer that those are the same thing.